### DISCUSSION TITLE
Fixed line validation not working

### DISCUSSION THREAD

**Discussion by farimir** [ysfkaya/filament-phone-input#30] [Q&A] [answered]
I am trying to validate my phone number as a fixed line number, however I cant get this to work.  I tried different options, but no luck so far.  Any suggestions?

This is my phone input:
![image](https://github.com/ysfkaya/filament-phone-input/assets/6318472/aec17611-726b-4b14-8194-75d005e6745d)

Relevant imports:
![image](https://github.com/ysfkaya/filament-phone-input/assets/6318472/310d6c3e-7c83-461d-b8da-40ea13a6199a)

Validation in general do work if I enter too many numbers for example.

**Comment by ysfkaya** [comment, marked as answer]
I just tagged a new release `2.2.1`. Now this method should work properly.

If it still not working, you can use the custom validation;

```php
->rule('phone:ZA,FIXED_LINE')
```

You can find more information [here](https://github.com/propaganistas/laravel-phone#validation)

**Reply by farimir** [reply]
Thank you very much!
